# name of first body
string body1_name

# name of second body
string body2_name

# flag to allow collisions or not
bool allow_collisions
